traffic_ops/app/lib/Extensions/TrafficStats/Builder/CacheStatsBuilder.pm

```diff
@@ -74,14 +74,12 @@ sub summary_query {
 	if ( $self->validate_keys() ) {
 
 		#'summary' section
-		my $query = sprintf(
-			'%s "%s" %s',
-			"SELECT mean(value), percentile(value, 5), percentile(value, 95), percentile(value, 98), min(value), max(value), sum(value), count(value) FROM",
-			$args->{series_name}, "WHERE cdn = '$args->{cdn_name}' AND
-			                             time > '$args->{start_date}' AND
-		                                 time < '$args->{end_date}' 
-		                                 GROUP BY time($args->{interval}), cdn"
-		);
+		my $query = qq[SELECT mean(value), percentile(value, 5), percentile(value, 95), percentile(value, 98), min(value), max(value), sum(value), count(value)
+				FROM "monthly"."$args->{series_name}.cdn.1min"
```

Comment:
we probably want a way to change the retention policy here.  Either through configs or args.  hard-coding to "monthly" could lead to issues.